src/items/implementations.md

```diff
@@ -50,10 +50,44 @@ bracketed set of associable items.
 The nominal type is called the _implementing type_ and the associable items are
 the _associated items_ to the implementing type.
 
-Inherent implementations associate the contained items to the implementing type.
-The associated item has a path of a path to the implementing type followed by
-the associate item's path component. Inherent implementations cannot contain
-associated type aliases.
+Inherent implementations associate the contained items to the
+implementing type.  Inherent implementations can contain [associated
+functions] (including methods), [associated types], and [associated
+constants]. They cannot contain associated type aliases.
+
+[associated functions]: associated-items.html#associated-functions-and-methods
+[associated types]: associated-items.html#associated-types
+[associated constants]: associated-items.html#associated-constants
+
+The path to an associated item is: any path to the implementing type,
+followed by the path to the associate item within the inherent
+implementation.
+
+Note that the path to the module containing the inherent
+implementation does not allow access to the associate item, unless the
+implementing type is re-exported from the same location.
+
+``` rust
+pub mod color {
+    pub struct Color(pub u8, pub u8, pub u8);
+}
+
+mod values {
+    use super::color::Color;
+    impl Color {
+        pub fn red() -> Color {
+            Color(255, 0, 0)
+        }
+    }
+}
+
+pub use self::color::Color;
+fn main() {
+    color::Color::red();  // actual path to the implementing type
+    Color::red();  // rexported paths to the implementing type also work
+    // bright_theme::Color::red();  // Does not work, because use in `theme` is not pub
```

Comment:
`bright_theme` doesn't exist. Did you mean `values` or did you want to rename module?

Reply:
whoops.  Changed the code sample too many times.  I was going to add another module with a pub use in it, but it ended up feeling like belaboring the point.

Reply:
fixed. and fixed the comment which mentioned yet another previous name for the secondary module.